src/services/todos.ts

```diff
@@ -0,0 +1,14 @@
+import todosFromServer from '../api/todos';
+import { Todo } from '../types/Todo';
+import { getUserById } from './user';
+
+export function getPreparedTodos() {
+  return todosFromServer.map(todo => ({
+    ...todo,
+    user: getUserById(todo.userId),
+  }));
+}
+
+export function getMaxId(todos: Todo[]) {
+  return Math.max(...todos.map(todo => todo.id));
```

Comment:
The `getMaxId` function does not handle the case where `todos` might be an empty array, which would result in `Math.max` returning `-Infinity`. Consider providing a default value, such as `0`, to handle this scenario safely.